src/models/core/monitor.rs

```diff
@@ -130,3 +130,10 @@ pub enum ScriptLanguage {
 	Python,
 	Bash,
 }
+
+/// Static mapping of script languages to their file extensions
+pub const LANGUAGE_EXTENSIONS: &[(&ScriptLanguage, &str)] = &[
```

Comment:
Maybe SCRIPT_LANGUAGE_EXTENSIONS for more context

Reply:
Pushed in #218